web/setup.py

```
@@ -163,6 +163,157 @@ def get_role(role: str):
 
 class ManageUsers:
 
+    @app.command()
+    @update_sqlite_path
+    def load_users(input_file: str,
+                   sqlite_path: Optional[str] = None,
```

Ruff (0.15.1)

[warning] Unused method argument: `sqlite_path`

(ARG002)